docs/clients/dovecot.txt

```diff
@@ -45,6 +45,25 @@ Configuring Dovecot to work with Virtual Exim
    Dovecot expects the select to return a set of columns with particular
    names, hence usage of the 'AS' keyword above.
 
-3. There's no step 3! Just restart Dovecot, and it should work.
+3. Exim can’t handle SHA512-crypt schemes for passwords, therefore you can use
+   the dovecot authenticator to login:
+   edit conf.d/10-master.conf
+     service auth {
+     ...
+     #SASL
+       unix_listener auth-client {
+       mode = 0660
+       user = vexim
+       }
+     ...
+     }
+
+   and edit conf.d/10-auth.conf
+     auth_mechanisms = plain login
+
+4. Edit your /etc/exim4/exim4.conf or /etc/exim4/conf.d/auth/01_exim4-config_vexim
```

Comment:
I think we should rename `01_exim4-config_vexim` to `30_vexim_authenticators`. I overlooked this in @Z3po's initial PR, but this current name doesn't make sense (`exim4-config` is the name of the Debian package).
It might also make sense to put your authenticators into a separate file (in which case the current file could be named `30_vexim_basic_auth` and your authenticators could go into `30_vexim_advanced_auth`, or something like that), but splitting is not necessary.